sympy/integrals/tests/test_manual.py

```diff
@@ -553,3 +552,19 @@ def test_quadratic_denom():
     assert manualintegrate(f, x) == 5*log(3*x**2 - 2*x + 8)/6 + 11*sqrt(23)*atan(3*sqrt(23)*(x - Rational(1, 3))/23)/69
     g = 3/(2*x**2 + 3*x + 1)
     assert manualintegrate(g, x) == 3*log(4*x + 2) - 3*log(4*x + 4)
+
+def test_issue_22757():
```

Comment:
The tests are more than are needed. It feels underwhelming to add a single test, but a single one would suffice to show that the issue has been addressed (and it is clear to the reviewer that this is not fixed for only one case). So maybe just the first test is sufficient.